src/librustc/diagnostics.rs

```diff
@@ -1641,13 +1641,13 @@ impl Foo {
 These attributes do not work on typedefs, since typedefs are just aliases.
 
 Representations like `#[repr(u8)]`, `#[repr(i64)]` are for selecting the
-discriminant size for C-like enums (when there is no associated data, e.g.
+discriminant size for enums with no data fields on any of the variants, e.g.
 `enum Color {Red, Blue, Green}`), effectively setting the size of the enum to
```

Comment:
You took out the opening parenthesis, but left the closing one here (after "`enum Color`").